oggm/core/flowline.py

```diff
@@ -983,6 +987,7 @@ def __init__(self, flowlines, mb_model=None, y0=0., glen_a=None,
         self.shapefac_stag = []
         self.flux_stag = []
         self.trib_flux = []
+        self.mb_on_surfaceh = []
```

Comment:
that is never used. was that intended for `mbs` in line 1014?

Reply:
Thanks! I overlooked this yes, needs removal